lib/common/strings.c

```diff
@@ -655,3 +655,10 @@ pcmk__parse_ll_range(const char *srcstring, long long *start, long long *end)
 
     return pcmk_rc_ok;
 }
+
+gboolean
+pcmk__str_in_list(GList *lst, const gchar *s)
+{
+    return lst && (g_list_find_custom(lst, "*", (GCompareFunc) strcmp) ||
+                   g_list_find_custom(lst, s, (GCompareFunc) strcmp));
```

Comment:
For efficiency, instead of searching the entire list for "\*", I'd check only whether the first item is "\*" (and maybe that list->next == NULL).

Reply:
Done.